web/src/beta/features/LeftPanelScene/index.tsx

```diff
@@ -0,0 +1,21 @@
+import SidePanel from "@reearth/beta/features/SidePanel";
+import SidePanelItem from "@reearth/beta/features/SidePanel/SidePanelItem";
+import SidePanelCard from "@reearth/beta/features/SidePanelCard";
+
+type Props = {};
+
+const LeftPanelScene: React.FC<Props> = () => {
+  return (
+    <SidePanel location="left">
+      <SidePanelItem>
+        <SidePanelCard title={"Outline"}>
+          {[...Array(100)].map((_, i) => (
+            <div key={i}>scrollable / {i}</div>
+          ))}
+        </SidePanelCard>
+      </SidePanelItem>
+    </SidePanel>
+  );
+};
+
+export default LeftPanelScene;
```

Comment:
I feel like this can be made reuseable so you don't have to have LeftPanelScene/Story/Widget/etc.

Can't you just add `location` and `title` props (and eventually the Array(100) boilerplate will be replaced by actual content that can be passed in as a children: ReactNode) and then have a single LeftPanel and single RightPanel component?

Reply:
I guess after implementing logic, especially store handling stuff, the component/hooks files which have content generation might get dirty/large.
So the purpose of separating components now was that each component will be able to have complex logic or props handling in the future.

But we don't have the actual code now, once tried to fix in this [commit](https://github.com/reearth/reearth/pull/463/commits/865dc9df9def25212960d6926484bf07fb03c5cf)

Reply:
Sorry the link to the commit is wrong 🙏🏻